modules/007-registrypackages/images/docker-centos/scripts/install

```diff
@@ -14,14 +14,30 @@
 # limitations under the License.
 
 set -Eeo pipefail
-PACKAGES_TO_INSTALL=""
-if ! rpm --quiet -q "$(rpm -qip docker-ce-cli.x86_64.rpm | grep Name | awk '{print $3}')"; then
-  PACKAGES_TO_INSTALL="docker-ce-cli.x86_64.rpm"
+packages_to_install=""
+package_docker_ce="docker-ce"
+package_file_docker_ce="${package}.x86_64.rpm"
+rpm_name_docker_ce="$(rpm -qip ${package_file_docker_ce} 2>/dev/null | grep Name | awk '{print $3}')"
+rpm_version_docker_ce="$(rpm -qip ${package_file_docker_ce} 2>/dev/null | grep Version | awk '{print $3}')"
+
+if ! rpm --quiet -q "${rpm_name_docker_ce}-${rpm_version_docker_ce}"; then
+  packages_to_install="${packages_to_install} ${package_file_docker_ce}"
+else
+  echo "RPM ${package_file_docker_ce} already installed."
 fi
-if ! rpm --quiet -q "$(rpm -qip docker-ce.x86_64.rpm | grep Name | awk '{print $3}')"; then
-  PACKAGES_TO_INSTALL="${PACKAGES_TO_INSTALL} docker-ce.x86_64.rpm"
+
+package_docker_ce_cli="docker-ce-cli"
+package_file_docker_ce_cli="${package}.x86_64.rpm"
```

Comment:
i am not looking for `package` variable, maybe here should be `package_file_docker_ce_cli`?

Reply:
yes it is error, thanks